packages/babel-preset-jsx/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@emotion/babel-preset-jsx",
```

Comment:
We should name this `@emotion/babel-preset-css-prop`. `jsx` is an implementation detail imo and could confuse people.

Reply:
Totally! That sounds so much better!